### PR TITLE
feat: stableSwap info page for multi-chain

### PR DESCRIPTION
**Pull request by Chef-Yogi**
---

## PR-Codex overview
This PR focuses on adding support for StableSwap subgraphs in the PancakeSwap codebase. 

### Detailed summary
- Added `STABLESWAP_SUBGRAPHS` constant in `chains/src/subgraphs.ts`
- Updated `BLOCKS_SUBGRAPH_CLIENT` constant to use `STABLESWAP_SUBGRAPHS` in `apps/web/src/config/constants/endpoints.ts`
- Added `STABLESWAP_SUBGRAPH_CLIENT` constant in `utils/graphql.ts`
- Updated `infoStableSwapClient` and `stableSwapClient` constants to use `STABLESWAP_SUBGRAPHS` in `utils/graphql.ts`
- Updated `fetchPoolData` function to handle StableSwap pools in `state/info/queries/pools/poolData.ts`
- Updated `InfoPageLayout` component to handle StableSwap submenus in `views/V3Info/components/Layout/index.tsx` and `views/Info/index.tsx`
- Updated `fetchDerivedPriceData` function to handle StableSwap info endpoints in `state/swap/fetch/fetchDerivedPriceData.ts`
- Updated `PoolPage` component to handle StableSwap fee display in `views/Info/Pools/PoolPage.tsx`
- Updated `BLOCKS_CLIENT` constants to include `BLOCKS_CLIENT_BASE` in `state/info/constant.ts`

> The following files were skipped due to too many changes: `apps/web/src/state/info/constant.ts`, `apps/web/src/state/info/queries/tokens/tokenData.ts`, `apps/web/src/state/info/queries/pools/topPools.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/config/constants/endpoints.ts [hunk 1]
```diff
@@ -32,6 +32,8 @@ export const BLOCKS_CLIENT_LINEA = BLOCKS_SUBGRAPHS[ChainId.LINEA]
 export const BLOCKS_CLIENT_BASE = BLOCKS_SUBGRAPHS[ChainId.BASE]
 export const BLOCKS_CLIENT_OPBNB = BLOCKS_SUBGRAPHS[ChainId.OPBNB]
 export const STABLESWAP_SUBGRAPH_CLIENT = 'https://api.thegraph.com/subgraphs/name/pancakeswap/exchange-stableswap'
+export const STABLESWAP_SUBGRAPH_CLIENT_ARBITRUM =
```
Comment (chefjackson) [inline apps/web/src/config/constants/endpoints.ts:35]: Let's move to chains

Comment (Chef-Yogi) [reply apps/web/src/config/constants/endpoints.ts:35]: ok

**Comment by chefjackson** [comment]
Remember to fix ts issues

**Comment by chefjackson** [comment]
@SocketSecurity ignore next@13.5.6

File: apps/web/src/config/constants/endpoints.ts (hunk 1 above)
Comment (chefjackson) [reply apps/web/src/config/constants/endpoints.ts:35]: Let's import from chains? Since we've added to chains

Comment (chefjackson) [inline apps/web/src/config/constants/endpoints.ts:35]: let's keep single source of truth. Use `STABLESWAP_SUBGRAPHS` from chains

Comment (Chef-Yogi) [reply apps/web/src/config/constants/endpoints.ts]: Done

**Review by chefjackson — APPROVED** [review]

File: apps/web/src/state/info/constant.ts [hunk 2]
```diff
@@ -108,6 +108,15 @@ export const multiChainQueryClient = {
   OPBNB: v2Clients[ChainId.OPBNB],
 }
 
+export const multiChainQueryStableClient = {
+  BSC: infoStableSwapClients[ChainId.BSC],
+  ARB: infoStableSwapClients[ChainId.ARBITRUM_ONE],
+}
+
+export const multiChanStableSwapStartBlock = {
+  ARB: 169319653,
```
Comment (chefjackson) [inline apps/web/src/state/info/constant.ts:117]: STABLESWAP_SUBGRAPHS_START_BLOCK

**Review by chefjackson — APPROVED** [review]

**Pull request merged**